examples/vue/src/styles.scss

```diff
@@ -15,6 +15,9 @@ body {
   padding: 0;
   margin: 0;
 }
+.amplify-icon[data-size='large'] {
+  height: 2rem !important;
```

Comment:
Were there issues with the icon that required this? Ideally we don't have any additional styles besides what the primitives provide.

Reply:
The chevron on the drop down for the select box was smaller then it was in React. So I added this to make it look exact. And for some reason the styles in the example app, don't automatically override the styles in the library.

Reply:
In the `amplify-ui/examples/next/pages/examples/listings/styles.scss` there is an overwrite to make the icon large-height to xxl 

`
  --amplify-components-icon-large-height: var(--amplify-font-sizes-xxl);
`

When I added this line into the Vue example styles sheet, it was getting overwritten by the package, so the chevron was smaller.

So added this line and just marked is as 2rem to match `--amplify-font-sizes-xxl` so it would look exact.

Reply:
@dbanksdesign ☝️ Something to consider as a default styling so icons look good by default.

Shouldn't `rem` be our default of `em`, also?

Reply:
`rem` is our default sizing unit: https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/tokens/fontSizes.js https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/tokens/components/icon.js

For changes like this in the future, we should propose those changes in the theme so we don't have style overrides in different places. I think bumping up the size of the icons makes sense in general so we should do it at the theme level. Could we open a new PR making that change in the theme and removing this override?